### PR TITLE
Set visibility of popovers, avoid trigger when hovering invisible pseudoelement

### LINKED ISSUE #297: Popover triggered by transparent pseudo-element
When hovering over the space occupied by the (transparent) popover pseudo-element, the popover itself is triggered to appear. This is visible in the official examples: https://www.getpapercss.com/docs/components/popovers/

Because there might be other content in that space, and it would be obscured by the popover, I think it'd be best to prevent this behavior. The fix could be something like...

```css
[popover-x]::after {
  visibility: hidden;
}
[popover-x]:hover::after {
  visibility: visible;
}
```

... with some additional handling for the opacity transition. Would it be OK for me to make a PR with these changes?

### PR DESCRIPTION
**Pull request by donmccurdy**
## Brief description

This PR is intended to fix #297, such that popovers do not appear when hovering over the area occupied by the (still invisible) popover. This can be important if other elements are beneath that area, and also [improves accessibility](https://css-tricks.com/snippets/css/toggle-visibility-when-hiding-elements/).

Unfortunately I'm having trouble getting the local development server running to test this change. In CONTRIBUTING.md I don't see anything about installing Hugo — do I need to do anything manually there? I'm on macOS. When running `npm run dev` I get errors like "node_modules/hugo-bin/vendor EACCES", or when installing Hugo manually and running ` hugo server --source=docs --disableFastRender` I get the error:

> `Configured defaultMarkdownHandler "blackfriday" not found. Did you mean to use goldmark? Blackfriday was removed in Hugo v0.100.0."`

I tried manually switching the markdown handler, Hugo runs and the site loads, but the demo PaperCSS components are missing from the site preview.

Thanks!

## Developer Certificate of Origin

- [x] I certify that these changes according to the Developer Certificate of Origin 1.1 as described at <https://developercertificate.org/>.

## Sample pictures

N/A

## Further details

- Fixes #297

### CODE REVIEW

File: src/components/_popovers.scss [hunk 1]
```diff
@@ -10,7 +10,8 @@
   &:hover {
     &::after {
       opacity: 1;
-      transition: opacity 0.2s ease-out;
+      transition: opacity 0.2s ease-out, visibility 0s linear 0s;
```
Comment (Oli8) [inline src/components/_popovers.scss:13]: `opacity 235ms ease-in-out` are the original transition values, probably better to stick to it.
The last `0s` is not needed as it is the default. 
```suggestion
      transition: opacity 235ms ease-in-out, visibility 0s linear;
```

File: src/components/_popovers.scss [hunk 2]
```diff
@@ -32,6 +33,8 @@
     text-align: center;
     top: -6px;
     transform: translateX(-50%) translateY(-100%);
+    transition: opacity 0.2s ease-out, visibility 0s linear 0.2s;
```
Comment (Oli8) [inline src/components/_popovers.scss:36]: Same as above, also you should remove the line with `@include transition(opacity);` as it is now overridden.
```suggestion
    transition: opacity 235ms ease-in-out, visibility 0s linear 0.2s;
```

**Review by Oli8 — CHANGES_REQUESTED** [review]

**Comment by donmccurdy** [comment]
Updates applied, thanks @Oli8!

**Comment by donmccurdy** [comment]
Oh, that's where the original value came from! I'd missed the scss macro yes, thanks. Fixed!

**Review by Oli8 — APPROVED** [review]

**Comment by Oli8** [comment]
:stop_sign:  Need someone allowed to merge to master and make a patch release

**Comment by rhyneav** [comment]
Thanks y'all! Just merged and published `v1.9.2"

**Pull request merged**